newscoop/src/Newscoop/NewscoopBundle/Security/Http/Authentication/FrontendLogoutSuccessHandler.php

```diff
@@ -50,4 +51,15 @@ public function onLogoutSuccess(Request $request)
 
         return parent::onLogoutSuccess($request);
     }
+
+    private function setTargetUrl(Request $request)
```

Comment:
it actually makes it clearer to read :) and makes it separated, not everything in one place